components/gitpod-cli/cmd/ports/ports-list.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package ports
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/supervisor"
+	"github.com/gitpod-io/gitpod/supervisor/api"
+	"github.com/spf13/cobra"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+func ListPortsCmd(cmd *cobra.Command, args []string) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	conn := supervisor.Dial()
+	client := api.NewStatusServiceClient(conn)
+
+	ports := supervisor.GetPortsList(ctx, client)
```

Comment:
Did you consider sorting the ports ascending before outputting them?

Reply:
Haven't considered it, but I agree it would be good for results consistency. Let me try it :)